### PR TITLE
Extract raw i18n initialization and function into its own module

### PR DESCRIPTION
**Pull request by noirbizarre**
Allows to use i18n outside Vue.js.

### CODE REVIEW

File: js/plugins/i18next.js [hunk 1]
```diff
@@ -1,60 +1,35 @@
...
-
-        // Make translations accesible globally and on instance
-        Vue._ = Vue.prototype._ = i18next.t;
-        Vue.lang = Vue.prototype.lang = lang;
-
-        log.debug('Plugin i18next loaded');
-    };
-
-});
+import * as i18n from 'i18n';
```
Comment (davidbgk) [inline js/plugins/i18next.js]: Isn't it possible to stay explicit on imports and avoid `*` given that you only use `_` and `lang`?

**Comment by davidbgk** [comment]
r+wc

File: js/plugins/i18next.js (hunk 1 above)
Comment (noirbizarre) [inline js/plugins/i18next.js]: It's not the same semantics as in Python.
This is the syntax to import the whole module as `i18n` where as in Python each module component is imported into the root.

Comment (noirbizarre) [inline js/plugins/i18next.js]: aka. this is equivalent to `import i18n` in Python.

**Pull request merged**